Tests/NimbleTests/AsynchronousTest.swift

```diff
@@ -203,9 +203,9 @@ final class AsyncTest: XCTestCase {
 #if !SWIFT_PACKAGE
         var executedAsyncBlock: Bool = false
         let asyncOperation: () -> Void = {
-            expect {
+            expect(expression: {
```

Comment:
What ambiguity is this resolving? (Between which two methods, I mean)

It doesn't seem like we want apps to have to make this change in their own tests, so is there another way we can eliminate the ambiguity (e.g. by deleting another deprecated API)?

Reply:
The ambiguity happens because of 2 `expect` functions with very close signature (see below) non of which is deprecated.
```
/// Make an expectation on a given actual value. The value given is lazily evaluated.
public func expect<T>(_ expression: @autoclosure @escaping () throws -> T?, file: FileString = #file, line: UInt = #line) -> Expectation<T> {...}

/// Make an expectation on a given actual value. The closure is lazily invoked.
public func expect<T>(_ file: FileString = #file, line: UInt = #line, expression: @escaping () throws -> T?) -> Expectation<T> {...}
```

Their implementation is identical. The only functional difference is `@autoclosure`.
Ambiguity complaint apparently is a blessing because then user must make a conscious choice and change call site accordingly.
In the case above compiler didn't complain about a thing. The test simply failed.